src/__stories__/00-Uncontrolled.test.tsx

```diff
@@ -1095,10 +1095,14 @@ describe('composed table rows', () => {
 
 describe('Nested object', () => {
   test('nested object table renders', () => {
-    const { getByRole } = render(<NestedObjectTable />);
+    const { getByRole,getByText } = render(<NestedObjectTable />);
```

Comment:
**The most minor thing in the world:** could we prettify this file? There seems to be no space between `,` and `getByText`.